doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<ul><li>In general: would it be better to have the "download model and upload to bucketFS" as a seperate Notebook and then mention in each model specific notebook "please download model &lt;model_name&gt; using &lt;the new download notbook&gt;." ? it seems to be a lot of duplication like this.</li><li>if this part is not moved i would add a subtitle headline before it.</li></ul><p> </p><ul><li>maybe add a link to "Huggingface Hub"?</li></ul><p> </p><ul><li>"Downloading the model to a local drive and subsequently uploading it into the BucketFS using CLI."</li></ul><p> </p><ul><li>"To demonstrate the "filling of a masked word"-task we will use the <a href="https://huggingface.co/StanfordAIMI/RadBERT" target="_blank">RadBERT model</a> which was pre-trained on radiology reports. "</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Made model loading a function in a separate notebook.</p>